pkg/config/seelog_config.go

```diff
@@ -0,0 +1,92 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package config
```

Comment:
can we move this to a new, separate package to isolate it more?
(for example, under `pkg/config/seelog`)

Reply:
Will do.